contracts/test/plugin/plugin.ts

```diff
@@ -7,29 +9,45 @@ import {
 import {erc165ComplianceTests} from '../helpers';
 import {osxCommonsContractsVersion} from '../utils/versioning/protocol-version';
 import {getInterfaceId, PluginType} from '@aragon/osx-commons-sdk';
+import {loadFixture} from '@nomicfoundation/hardhat-network-helpers';
 import {SignerWithAddress} from '@nomiclabs/hardhat-ethers/signers';
 import {expect} from 'chai';
 import {ethers} from 'hardhat';
 
-describe('Plugin', function () {
-  let deployer: SignerWithAddress;
-  let plugin: PluginMockBuild1;
-
-  before(async () => {
-    [deployer] = await ethers.getSigners();
-    plugin = await new PluginMockBuild1__factory(deployer).deploy(
-      ethers.constants.AddressZero
-    );
-  });
+type FixtureResult = {
+  deployer: SignerWithAddress;
+  plugin: PluginMockBuild1;
+  daoMock: DAOMock;
+  Build1Factory: PluginMockBuild1__factory;
+};
+
+async function fixture(): Promise<FixtureResult> {
+  const [deployer] = await ethers.getSigners();
+
+  const Build1Factory = new PluginMockBuild1__factory(deployer);
+  const daoMock = await new DAOMock__factory(deployer).deploy();
 
+  const plugin = await Build1Factory.deploy(daoMock.address);
+
+  return {
+    deployer,
+    plugin,
+    daoMock,
+    Build1Factory,
+  };
+}
```

Comment:
move either to the bottom or in the other files to the top so that the fixtures are always in the same location

Reply:
Done in 3d156e7e6accd773f4bc1a0af210556f083dc251